lib/action_account.rb

```diff
@@ -49,6 +49,12 @@ def subtask(name)
   end
 
   class ReviewReject
+    def log_message(uuid, log, table, messages)
```

Comment:
I'd migrate these to keyword arguments so it's harder to send in the wrong order
```suggestion
    def log_message(uuid:, log:, table:, messages:)
```

Reply:
Implemented!